src/main/java/seedu/commando/logic/commands/CommandFactory.java

```diff
@@ -276,10 +287,19 @@ private Command buildEditCommand() throws IllegalValueException {
             x -> command.dueDate = x
         );
 
-        // Extract title
-        sequentialParser.extractText().ifPresent(title -> {
-            command.title = new Title(title);
-        });
+        // Try to extract title, if there was no quoted title
+        // Otherwise, use the quoted title
+        if (quotedTitle.isPresent()) {
+            command.title = new Title(quotedTitle.get());
+        } else {
+            sequentialParser.extractText().ifPresent(
+                title -> command.title = new Title(title)
+            );
+        }
+
+        if (!sequentialParser.isInputEmpty()) {
```

Comment:
isInputEmpty() will result in throwing an exception with messages.invalid command format, should it be called isInputFormatValid or something to make it consistent?